modules/odc-importing-codebase-from-git-to-create-application.adoc

```diff
@@ -51,7 +52,7 @@ The *Serverless Deployment* option is displayed in the *Import from git* form on
 
 . In the *Pipelines* section, select *Add Pipeline*, and then click *Show Pipeline Visualization* to see the pipeline for the application.
 
-. In the *Advanced Options* section, the *Create a route to the application* is selected by default so that you can access your application using a publicly available URL. You can clear the check box if you do not want to expose your application on a public route.
+. In the *Advanced Options* section, the *Target port* and the *Create a route to the application* is selected by default so that you can access your application using a publicly available URL on a specific target port. You can change the target port if you want to expose your application on a different port. You can clear the check box if you do not want to expose your application on a public route.
```

Comment:
The target port doesn't change the external "public available URL" port. It changes the port where the application (container) exposes their data and which will be then connected/proxied by the Service.

Based on your change I would recommend something like this:

```suggestion
. In the *Advanced Options* section, the *Target port* and the *Create a route to the application* is selected by default so that you can access your application using a publicly available URL. You can change the target port if your application uses a different port to expose their data. You can clear the check box if you do not want to expose your application on a public route.
```